service/routes.py

```diff
@@ -147,25 +147,16 @@ def delete_wishlists(wishlist_id):
   )
 
 ######################################################################
-# Delete products from a wishlist
+# Delete all products from a wishlist
 ######################################################################
 @app.route("/wishlists/<int:wishlist_id>/products", methods=["DELETE"])
 def delete_products_from_wishlist(wishlist_id):
   """
-  Delete products from wishlist
-  This endpoint will delete products from the wishlist with id specified in the URL.
+  Delete all products from a wishlist
+  This endpoint will delete all products from the wishlist with id specified in the URL.
```

Comment:
Now we are removing all products regardless of what's in the body, so could you remove this line in the comment since it is no longer relevant?